projects/quantify.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Quantify
+emoji: 🚀
+project_url: https://gitlab.com/quantify-os
```

Comment:
```suggestion
project_url: "https://gitlab.com/quantify-os"
```

Reply:
@crazy4pi314 check about this

Reply:
The quotes are not invalid, but not needed